services/queue/schemas/v1/task-completed-message.yml

```diff
@@ -6,6 +6,15 @@ type:         object
 properties:
   version:    {$const: message-version}
   status:     {$ref: 'task-status.json#'}
+  task:
+    description: |
+      Subset of a task definition
+    type:               object
+    properties:
+      tags:     {$ref: "task.json#/properties/tags"}
+    additionalProperties: false
+    required:
+      - tags
```

Comment:
Since this hunk is repeated three times, let's pull it out into its own schema file, and also add some context to the description, perhaps:

> Subset of a task definition containing values that are useful for determining whether a message is interesting to the receiver.  Where the full task definition is required for a "interesting" task, the receiver should call `queue.task` to download that definition.